<?xml version="1.0" ?>
<launch>

  <!-- <include file="$(find feature_extraction)/launch/osm.launch" ns="feature_extraction">
  </include> -->

  <node name="maplite" pkg="topomap_localizer" type="maplite_node" ns="maplite" output="screen">
    
    <!--  FRB  -->
    <param name="tx" value="32" />  
    <param name="ty" value="-33.6" />  
    <param name="theta" value="-2.25" />

    <param name="particleNum" value="100" />
    <param name="nearest_node_number" value="50" />

    <param name="noise_coefficient_action_x" value="0.7" />
    <param name="noise_coefficient_action_y" value="0.7" />
    <param name="noise_coefficient_action_t" value="0.2" />
    <param name="noise_stdev_action" value="1.0" />
    
    <param name="noise_coefficient_init_x" value="1.0" />
    <param name="noise_coefficient_init_y" value="1.0" />
    <param name="noise_coefficient_init_t" value="0.1" />
    <param name="noise_stdev_init" value="4.0" />

    <param name="noise_coefficient_resample_x" value="0.5" />
    <param name="noise_coefficient_resample_y" value="0.5" />
    <param name="noise_coefficient_resample_t" value="0.05" />
    <param name="noise_stdev_resample" value="1.0" />

    <param name="center_rec_x" value="8.0" />
    <param name="center_rec_y" value="-28.0" />

    <param name="threshold_odom_init" value="20" />
    <param name="ratio_resample" value="1.0" />
    <param name="ratio_calpost" value="1.0" />

  </node>

</launch>
